core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -176,11 +176,11 @@ void create(String tableName, NewTableConfiguration ntc)
    *
    * @param tableName
    *          Name of a table to create and import into.
-   * @param importDir
-   *          Directory that contains the files copied by distcp from exportTable
+   * @param importDirs
+   *          A set of directories containing the files copied by distcp from exportTable
    * @since 1.5.0
    */
-  void importTable(String tableName, String importDir)
+  void importTable(String tableName, Set<String> importDirs)
```

Comment:
This is public API and cannot change without a deprecation cycle.
A new API is needed, which should co-exist with this one, until this one is removed.
I didn't look at the rest of this PR, since this is a blocker right away.

Reply:
I took a crack at this in 7f241a70c3be2d0d5c5b4159684fc5d6646bdcc7 - let me know if this is sufficient.